ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -4,4 +4,45 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-# TODO(skuznets): oh boy
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+export SSH_PRIV_KEY_PATH=${cluster_profile}/ssh-privatekey
+export PULL_SECRET_PATH=${cluster_profile}/pull-secret
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+export HOME=/tmp
+
+case "${CLUSTER_TYPE}" in
+aws) export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred;;
+azure4) export AZURE_AUTH_LOCATION=${cluster_profile}/osServicePrincipal.json;;
+gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${cluster_profile}/gce.json;;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
+
+mkdir "${ARTIFACT_DIR}/installer/"
+cp "${SHARED_DIR}/install-config.yaml" "${ARTIFACT_DIR}/installer/"
+
+# move private key to ~/.ssh/ so that installer can use it to gather logs on
+# bootstrap failure
+mkdir -p ~/.ssh
+cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
+
+# TODO RELEASE_IMAGE_INITIAL / upgrade tests
+# TODO mirror variant
+# TODO manual override
+
+TF_LOG=debug openshift-install --dir="${ARTIFACT_DIR}/installer" create cluster 2>&1 | grep -v password &
```

Comment:
@bbguimaraes this is what has to change. When the final steps run `openshift-install destroy cluster` they will clean up the directory to allow only the necessary bits to remain. The installer should run in `${SHARED_DIR}/installer` and we should opt to copy into "${ARTIFACT_DIR}" as an opt-in thing after.

Reply:
Oh, sorry, I thought you were questioning this:

>> And [I don't see](https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_release/6708/rehearse-6708-pull-ci-openshift-installer-master-e2e-steps/5/artifacts/e2e-steps/) assets from the ipi-deprovision-deprovision container? Or logs from any of the step containers?
> 
> Logs go to a [separate directory](https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_release/6708/rehearse-6708-pull-ci-openshift-installer-master-e2e-steps/5/logs/) as in template tests.

The issue I pointed out is still valid, although I don't see it as blocking for this PR.

Reply:
Yes, the difference between the template and multi-stage tests is that for the former, `openshift-install destroy cluster` will modify the `installer` directory in the shared volume.

It looks like the only file left after the cluster is destroyed is the installation log:

https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/integr8ly_integreatly-operator/318/pull-ci-integr8ly-integreatly-operator-master-e2e/921/artifacts/e2e/installer/

If we want the same behavior in multi-stage, we can copy just that file at the end instead of pointing the installation at the artifact directory. I'll make that change.

Reply:
https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_release/6708/rehearse-6708-pull-ci-openshift-installer-master-e2e-steps/8/artifacts/e2e-steps/ipi-install-install/